Tasks/GradleV2/task.json

```diff
@@ -16,7 +16,7 @@
     "version": {
         "Major": 2,
         "Minor": 143,
-        "Patch": 1
```

Comment:
Please run the build locally as well. It will add the version change to the loc file.